### PR TITLE
Make 3.7 branch to use GHA

### PR DESCRIPTION
**Pull request by goosemania**
[noissue]

### CODE REVIEW

File: .github/workflows/scripts/script.sh [hunk 1]
```diff
@@ -4,35 +4,49 @@
...
-if [ "$TEST" = "docs" ]; then
+if [[ "$TEST" = "docs" || "$TEST" = "publish" ]]; then
   cd docs
   make PULP_URL="http://pulp" html
   cd ..
 
+  echo "Validating OpenAPI schema..."
+  cat $PWD/.ci/scripts/schema.py | cmd_stdin_prefix bash -c "cat > /tmp/schema.py"
+  cmd_prefix bash -c "python3 /tmp/schema.py"
+  # cmd_prefix bash -c "pulpcore-manager spectacular --file pulp_schema.yml --validate"
```
Comment (fao89) [inline .github/workflows/scripts/script.sh:37]: I can see you got the right version

**Review by fao89 — APPROVED** [review]
the 2 things I remembered could break the CI were the OpenAPI test and the app label change, both were right here, not sure if have other things that may break a release

**Comment by goosemania** [comment]
@fao89 , it could also be the docs script publishing in the wrong place.
But it seems that this one should work. It worked for the container plugin. Codepaths are a bit different for pulpcore but I think it should be fine.

**Comment by fao89** [comment]
https://github.com/pulp/pulpcore/blob/master/functest_requirements.txt#L6

**Comment by goosemania** [comment]
> https://github.com/pulp/pulpcore/blob/master/functest_requirements.txt#L6

test_bindings.* were missing as well. We'll see what now :)

**Review by mdellweg — APPROVED** [review]

**Pull request merged**